core/src/main/java/edu/wpi/grip/core/events/SourceStartedEvent.java

```diff
@@ -0,0 +1,19 @@
+package edu.wpi.grip.core.events;
+
+
+import edu.wpi.grip.core.Source;
+
+/**
+ * An event that occurs when a source is started.
+ */
+public class SourceStartedEvent {
+    private final Source source;
+
+    public SourceStartedEvent(Source source){
```

Comment:
Also, I think it's confusing that `StopStartSource` is a thing, yet you can have started and stopped events for *any* source.